lib/rules/generator-star-spacing.js

```diff
@@ -135,16 +139,14 @@ module.exports = {
                 const after = leftToken.value === "*";
                 const spaceRequired = modes[kind][side];
                 const node = after ? leftToken : rightToken;
-                const type = spaceRequired ? "Missing" : "Unexpected";
-                const message = "{{type}} space {{side}} *.";
-                const data = {
-                    type,
-                    side
-                };
+                const messageId = spaceRequired ? "missing" : "unexpected";
+
+                // const message = "{{type}} space {{side}} *.";
+                const data = { side };
```

Comment:
I'd like to separate message IDs for `before` / `after` because languages other than English can't use the word as is.